tests/lib/Output/Generator/_fixtures/Json_FieldTypeHashGeneratorTest__testGenerateUsingNormalizer.out

```diff
@@ -0,0 +1 @@
+{"Field":{"fieldValue":{"id":1,"type":"foo","internal_hash":{"foo":"bar"},"internal_list":[42,56]}}}
```

Comment:
Missing new line at EOF

Reply:
Can't do that. Those files are generated by tests themselves and require strict equality.

If I would to add a newline, I would need to adjust all the other test snapshots (for some reason called fixtures here, but that's beside the point :) )

Reply:
> Can't do that. Those files are generated by tests themselves and require strict equality.
> 
> If I would to add a newline, I would need to adjust all the other test snapshots (for some reason called fixtures here, but that's beside the point :) )

Meh, ok. Let's avoid it in the future.